reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -191,9 +198,13 @@ public Emission getReason() {
 	}
 
 	/**
-	 * A handler for any non-successful emission result from operations
-	 * such as {@link Many#emitNext(Object, EmitFailureHandler)}
-	 * that allows retrying or failing the operation.
+	 * A handler supporting the emit API (eg. {@link Many#emitNext(Object, EmitFailureHandler)}),
+	 * checking non-successful emission results from underlying {@link Many#tryEmitNext(Object) tryEmit}
+	 * API calls to decide whether or not such calls should be retried.
+	 * Other than instructing to retry, the handler are allowed to have side effects
+	 * like parking the current thread for longer retry loops. They don't however influence the
```

Comment:
not 100% sure about this one, but:

```suggestion
	 * like parking the current thread for longer retry loops. They don't, however, influence the
```